cpython-unix/build.py

```diff
@@ -88,6 +88,11 @@ def add_target_env(env, build_platform, target_triple, build_env):
     extra_host_cflags = []
     extra_host_ldflags = []
 
+    # Add compiler-rt for aarch64-musl to resolve missing builtins
```

Comment:
Honestly I don't understand why we need to be explicit about this.

Without this, the bzip2 and openssl builds fail with missing symbols (it's possible more do too, I didn't play whackamole for long).